dotnet/samples/GettingStartedWithProcesses/Step04/Plugins/CalendarPlugin.cs

```diff
@@ -0,0 +1,151 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+using System.ComponentModel;
+using System.Text.Json.Serialization;
+using Microsoft.SemanticKernel;
+
+namespace Step04.Plugins;
+
+internal sealed record CalendarEvent(
+    string Title,
+    string Start,
+    string? End,
+    string? Description = null)
+{
+    [JsonIgnore]
+    public DateTime StartDate { get; } = DateTime.Parse(Start);
```

Comment:
Maybe add CultureInfo.InvariantCulture here?

Reply:
Yes, always best practice to include `CultureInfo`; although, I might favor `CultureInfo.CurrentCulture`?